app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render partial: "admin/admin_nav" %>
+
+<br><br>
+
+<br><br><h2 style="background-color:lightgrey; border-top: 3px solid black; padding:10px; margin:20px;"><center>Invoices</h2>
```

Comment:
I didn't know you could chain the `<br>`'s here

Reply:
I found out last night and it works a charm.